packages/monaco-graphql/src/GraphQLWorker.ts

```diff
@@ -94,8 +97,9 @@ export class GraphQLWorker {
         range: toMonacoRange(
           getRange(
             {
-              column: graphQLPosition.character,
-              line: graphQLPosition.line,
+              // we add one here because `getRange()` expects a 1-indexed position
```

Comment:
just a nit but this would be very useful to have in this file as `toMonacoPosition()` in `../utils`? I think it's the only place where we need to call it.
https://github.com/graphql/graphiql/blob/0f38050953a4158170034b94429c0429a229e653/packages/monaco-graphql/src/utils.ts#L42